README.md

```diff
@@ -2,15 +2,21 @@
 
 This Helm repository contains the default Helm charts for the Okteto Cloud default repository:
 
+- [CockroachDB](cockroachdb/README.md)
+- [Drupal](drupal/README.md)
+- [Flux](flux/README.md)
 - [Hello World](hello-world/README.md)
+- [Inlets](inlets/README.md)
 - [MariaDB](mariadb/README.md)
-- [Movies](movies/README.md)
 - [MongoDB](mongodb/README.md)
+- [Movies](movies/README.md)
 - [OpenFaaS](openfaas/README.md)
 - [PostgreSQL](postgresql/README.md)
 - [RabbitMQ](rabbitmq/README.md)
 - [Redis](redis/README.md)
-- [WordPress](wordpress/README.md)
```

Comment:
Don't remove wordpress, it will be enable soon

Reply:
you can edit it!

Reply:
that's why I added another commit to highlight it

Reply:
But you can deploy it from this repo with `helm install` although it is disabled in the UI.